src/mame/seta/seta.cpp

```diff
@@ -9416,62 +9416,62 @@ Other: Actel A1020A PLC84C (used for graphics and graphic rom decode)
 */
 
 ROM_START( wiggie )
-	ROM_REGION( 0x40000, "maincpu", 0 ) /* 68000 Code */
+	ROM_REGION( 0x40000, "maincpu", 0 ) // 68000 Code
 	ROM_LOAD16_BYTE( "wiggie.e19", 0x00000, 0x10000, CRC(24b58f16) SHA1(96ef92ab79258da9322dd7e706bf05ac5143f7b7) )
 	ROM_LOAD16_BYTE( "wiggie.e21", 0x00001, 0x10000, CRC(83ba6edb) SHA1(fa74fb39599ed877317db73d02d14df5b475fc35) )
 
-	ROM_REGION( 0x40000, "audiocpu", 0 ) /* sound cpu code */
+	ROM_REGION( 0x40000, "audiocpu", 0 ) // sound CPU code
 	ROM_LOAD( "wiggie.a5", 0x00000, 0x10000, CRC(8078d77b) SHA1(4e6855d396a1bace2810b13b7dd08ccf5de89bd8) )
 
-	ROM_REGION( 0x040000, "oki", 0 ) /* Samples */
+	ROM_REGION( 0x040000, "oki", 0 ) // Samples
 	ROM_LOAD( "wiggie.d1", 0x00000, 0x40000, CRC(27fbe12a) SHA1(73f476a03b321ed1ae89104f5b32d77153fabb82) )
 
 	ROM_REGION( 0x80000, "gfx1", 0 )
-	ROM_LOAD( "wiggie.j16", 0x00000, 0x20000, CRC(4fb40b8a) SHA1(120c9fd677071485a9f8accc2385117baf542b9c) ) /* Drawn nude girls for the backgrounds */
+	ROM_LOAD( "wiggie.j16", 0x00000, 0x20000, CRC(4fb40b8a) SHA1(120c9fd677071485a9f8accc2385117baf542b9c) ) // Drawn nude girls for the backgrounds
 	ROM_LOAD( "wiggie.j18", 0x20000, 0x20000, CRC(ebc418e9) SHA1(a9af9bebce56608b0533d7d147191ebdceaca4e4) )
 	ROM_LOAD( "wiggie.j20", 0x40000, 0x20000, CRC(c073501b) SHA1(4b4cd0fed5efe12bcd10f98a71becc212e7e753a) )
 	ROM_LOAD( "wiggie.j21", 0x60000, 0x20000, CRC(22f6fa39) SHA1(d3e86e156434153335c5d2ce71417f35097f5ab7) )
 ROM_END
 
-ROM_START( superbar ) /* All roms had a "PROMAT" label with no other information.  Rom size was silkscreened on the PCB */
-	ROM_REGION( 0x40000, "maincpu", 0 ) /* 68000 Code */
+ROM_START( superbar ) // All roms had a "PROMAT" label with no other information. ROM size was silkscreened on the PCB
+	ROM_REGION( 0x40000, "maincpu", 0 ) // 68000 Code
```

Comment:
There’s no reason to remove the extra space between sentences in the comment here.

Reply:
Changed back, thx